test/toHtml.js

```diff
@@ -14,7 +14,7 @@ describe('toHtml', function () {
             .on('data', function (file) {
                 expect(file.contents + '').to.be.equal('<h1 class="page">Hello, world!</h1>');
             })
-            .on('error', done)
+            .on('error', function() {})
```

Comment:
but why?

Reply:
looks like there was an error here: https://github.com/gulp-bem/gulp-bem-xjst/pull/117/commits/f7b540914dff2a92e366fbdf9c60bdbc43b539d7#diff-168726dbe96b3ce427e7fedce31bb0bcR118

fixed it and done returns